src/exception_summary.jl

```diff
@@ -0,0 +1,98 @@
+###############
+#=
+# TODOs
+- Call it from our codebase
+- Unit tests
+- Seen set, for deduplication
+=#
+
+# Consider adding a _summarize_exception() overload for DistributedException
+#     Pros: less noise
+#     Cons: possibly hiding intermediate exceptions that might have been helpful to see.
+
+const TITLE = "=== THE NEEDLE ==="
```

Comment:
😁 as much as i do agree this was fun, i'm going to change this to something that is more obvious when just reading the log message 😏  ❤️ haha sorry
```suggestion
const TITLE = "=== EXCEPTION SUMMARY ==="
```

Reply:
😭